### DISCUSSION TITLE
Faster than distilled whisper? Could run distilled models?

### DISCUSSION THREAD

**Discussion by OliverBaumgart** [ggml-org/whisper.cpp#2298] [Q&A]
Hello,
was wondering if this is faster than the distilled models from https://huggingface.co/distil-whisper ? Or could it even potentionally use those distilled models for even faster processing ?

Thanks!

**Comment by mrfragger** [comment]
only distill-large-v3 is usable with whisper.cpp I get 6.0x to 8.3x realtime speed.  It's amazing.  English only though.

Whisper distil-large-v3 model transcribed at 8.36x realtime speed
Set language: en | Translated: 
Max length: 120 | Print Colors: 
No Graphics:  | Flash Attention: -fa
